### DISCUSSION TITLE
How do I re-assign a const inside script setup tag?

### DISCUSSION THREAD

**Discussion by dannyvalle** [nuxt/nuxt#17664] [General]
I am new/learning Nuxt. In my script tag, I am using fetch to retrieve data via an api. In the example below I get back a single entry. The object returned has a key of "data":

```
{
      "data": [{
                  "id": 1,
                  "title": "Article 1",
                  "description": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis perspiciatis quasi laborum vel velit! Enim minima distinctio cumque provident debitis consequatur aspernatur esse? Quasi, corrupti aliquam est in dignissimos iste?"
            }
}
```
In order for me to output the title for example, I have to write this: {{ data.data.title }}

I would like to re-assign to a const so I can just write this: {{ entry.title }}. See example below:

```
<script setup>
	const { data } = await useFetch('https://examplesite.com/items/articles')
	const entry = data.data
</script>
```

Now when I use {{ entry.title }}, no text is outputted. Just an empty string. 

How do I properly do this?

**Comment by kilobyte2007** [comment]
Not sure how is this Nuxt related but it seems that your data is actually an array of objects so you have to use:
```
const entry = data.data[0]
```

**Comment by gvalex13** [comment]
useFetch returns already reactive Vue variable. Try this
`const entry = data.value.data`

**Reply by dannyvalle** [reply]
`const entry = data.value.data`

That worked, thank you!
